src/features/create-quiz/create-quiz-second-card/components/answer-item.tsx

```diff
@@ -20,27 +24,41 @@ export const AnswerItem: React.FC<AnswerItemProps> = ({
   index,
   handleClick,
   handleChange,
+  getError,
+  formValidation,
+  handleRadioChange,
 }) => {
   return (
     <div className="create-quiz__answer">
-      <div className="create-quiz__input-label">Answer {`0${index + 1}`}</div>
+      <label htmlFor="question" className="create-quiz__input-label">
+        Answer {`0${index + 1}`}*
```

Comment:
Is this displayed well if there are more than 9 answers?

Reply:
I changed it now, and it will be displayed well.